MekHQ/unittests/mekhq/campaign/mission/AtBDynamicScenarioFactoryTest.java

```diff
@@ -0,0 +1,141 @@
+package mekhq.campaign.mission;
+
+import megamek.common.Entity;
+import megamek.common.Game;
+import megamek.common.MekSummary;
+import megamek.common.MekSummaryCache;
+import megamek.common.Player;
+import megamek.common.enums.SkillLevel;
+import mekhq.campaign.Campaign;
```

Comment:
Copyright Header

Reply:
I should just write a test I can run locally, save us all some time -_-;